metricbeat/module/kubernetes/state_cronjob/_meta/docs.asciidoc

```diff
@@ -1,6 +1,5 @@
 This is the `state_cronjob` metricset of the Kubernetes module.
 
-This metricset does not add metadata by default and hence in order to
-add metadata for this one need to configure the metricset with `add_metadata: true`
-and uncomment the proper `apiGroup` in the `ClusterRole`. Metadata are only available
-for versions of k8s >= v1.21.
+This metricset adds metadata by default only for versions of k8s >= v1.21.
+For older versions the APIs are not compatible and one need to configure the
+metricset with `add_metadata: false` and remove the proper `apiGroup` in the `ClusterRole`.
```

Comment:
Maybe we should be a bit more specific here of what they need to remove. Why not add an example with the specific part of the cluster role they need to remove?